packages/common-ts/src/base-service/validators.ts

```diff
@@ -49,6 +49,14 @@ const logLevel = makeValidator<LogLevel>((input) => {
   }
 })
 
+const address = makeValidator<string>((input) => {
+  if (!ethers.utils.isHexString(input, 20)) {
+    throw new Error(`expected input to be an address: ${input}`)
+  } else {
+    return input as string
```

Comment:
```suggestion
    return input as `0x${string}`
```
Cast it as this so the type works in wagmi/viem too

Reply:
Updated